<launch>
  <arg name="device_name"          default="/dev/ttyUSB0"/>
  <arg name="baud_rate"            default="1000000"/>

  <arg name="scan_range"           default="10"/>

  <param name="device_name"        value="$(arg device_name)"/>
  <param name="baud_rate"          value="$(arg baud_rate)"/>

  <param name="scan_range"         value="$(arg scan_range)"/>

  <node name="open_manipulator_dynamixel_controller" pkg="open_manipulator_dynamixel_ctrl" type="dynamixel_controller" required="true" output="screen"/>
</launch>
